cmake/utilities.cmake

```diff
@@ -82,7 +82,6 @@ endfunction()
 
 # Generates the global_c_settings, global_cxx_settings targets and the respective thirdparty variant
 function(generateGlobalSettingsTargets)
```

Comment:
Moving the flags in a separate files would be helpful; additionally, the targets being used right now against the third party libraries are leaking osquery-specific definitions. I changed some of those interface targets but they could use a real refactor

Reply:
As a heads up, I'm working on this.